app.js

```diff
@@ -0,0 +1,30 @@
+import isYes from './is-yes.js';
```

Comment:
Everything looks good, just try to leave blank spaces between the sections you noted so it's easier to read.

Reply:
Looking back I can see how the comments made that a lot harder to read, thank you for the feedback.